doc/newsfragments/2513_new.added_errorBoundary.rst

```diff
@@ -0,0 +1 @@
+Wrapping main webUI components in Error Boundary to prevent unexpected UI crash.
```

Comment:
Can you please give an example of such a crash (say most commonly observed by you) so that users can relate?

Reply:
Also use either "error boundary" if referencing the concept or "ErrorBoundary" if it is the React component referenced.

Reply:
I am referring to the name of the concept and tried to emphasize it, hence the capitals. Going to rephrase then anyway.